website/content/en/docs/getting-started-with-terraform/_index.md

```diff
@@ -0,0 +1,392 @@
+
+---
+title: "Getting Started with Terraform"
+linkTitle: "Getting Started with Terraform"
+weight: 10
+---
+
+Karpenter automatically provisions new nodes in response to unschedulable
+pods. Karpenter does this by observing events within the Kubernetes cluster,
+and then sending commands to the underlying cloud provider.
+
+In this example, the cluster is running on Amazon Web Services (AWS) Elastic
+Kubernetes Service (EKS). Karpenter is designed to be cloud provider agnostic,
+but currently only supports AWS. Contributions are welcomed.
+
+This guide should take less than 1 hour to complete, and cost less than $0.25.
+Follow the clean-up instructions to reduce any charges.
+
+## Install
+
+Karpenter is installed in clusters with a helm chart.
+
+Karpenter additionally requires IAM Roles for Service Accounts (IRSA). IRSA
+permits Karpenter (within the cluster) to make privileged requests to AWS (as
+the cloud provider).
+
+### Required Utilities
+
+Install these tools before proceeding:
+
+1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html)
+2. `kubectl` - [the Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+3. `eksctl` - [the CLI for AWS EKS](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+4. `helm` - [the package manager for Kubernetes](https://helm.sh/docs/intro/install/)
```

Comment:
Need to add tf and remove eksctl

Reply:
Doh! Missed that spot. Just pushed the update. Feel free to suggest another tagline/description for Terraform. I mostly took a snippet from the TF landing page.

Reply:
works for me!